tiledb/sm/array/array_directory.h

```diff
@@ -321,6 +322,62 @@ class ArrayDirectory {
   /*                API                */
   /* ********************************* */
 
+  /**
+   * Loads the schema of a schema uri from persistent storage into memory.
+   *
+   * @param array_schema_uri The URI path of the array schema.
+   * @param encryption_key The encryption key to use.
+   * @return Status, the loaded array schema
+   */
+  static tuple<Status, optional<shared_ptr<ArraySchema>>>
```

Comment:
I'm inclined to change these to throw here as well, since there's only 7 call sites touched in this PR

Reply:
Fixed.